app/modules/notifications/models.py

```diff
@@ -263,6 +269,7 @@ class NotificationChannel(str, enum.Enum):
             'deadline',
         },
         'allow_multiple': True,
+        'subtype_of': NotificationType.individual_merge_all,
```

Comment:
I like the idea of subtype of but think it should be a list for future flexibility. 
E.g show me all collab notifications, show me all collab requets, show me all collab declines, show me all collab confirms. Then many of the notifs would have multiple subtypes.

Reply:
implemented